config/examples/Creality/CR-10 V3/Configuration.h

```diff
@@ -1567,7 +1567,7 @@
 /**
  * Auto-leveling needs preheating
  */
-//#define PREHEAT_BEFORE_LEVELING
+#define PREHEAT_BEFORE_LEVELING
```

Comment:
For improved accuracy, but it can be confusing as you don't get any clear message that the printer is waiting for heating

Reply:
I changed this back to commented as original.
 - You don't have a clear message that the print is waiting for heating before ABL
 - The heating is done after homing so I'm not sure it's good for accuracy if the mesh is populated from differences from homing values. Probably better to manually pre heat.

Reply:
> You don't have a clear message that the print is waiting for heating before ABL

Everything is a work-in-progress. Please contribute to improve this facet!